### DISCUSSION TITLE
How to add usefull commands to the install script like dockers LXC

### DISCUSSION THREAD

**Discussion by holland-built** [tteck/Proxmox#2782] [Q&A]
Hello.   Would it be possible to add the option to ask on the VM installs for update/ upgrade and then ask for SSH, guest agent, docker, docker compose, dockge, and portainer.

**Comment by tteck** [comment]
The Docker Lxc has options for Portainer, Portainer Agent, and Docker Compose. For an Ubuntu VM, https://github.com/tteck/Proxmox/discussions/2072 or a Dbian VM, https://github.com/tteck/Proxmox/discussions/1988

**Reply by holland-built** [reply]
I thank you for the that.  The images I need to run after install cant  be docker in docker.

**Reply by tteck** [reply]
Understand what you need before creating a new instance.

**Comment by holland-built** [comment]
Yes sir I understand I just thought I might be an easy add on for the full Os.   Have a great evening    And appreciate the smokeping
